OrbitGl/TextRenderer.cpp

```diff
@@ -270,6 +270,85 @@ void TextRenderer::AddText( const char* a_Text
     AddTextInternal( m_Font, a_Text, ColorToVec4( a_Color ), &m_Pen, a_MaxSize, a_Z );
 }
 
+//-----------------------------------------------------------------------------
+void TextRenderer::AddTextTrailingTimePrioritized( const char* a_Text
+                                   , float a_X
+                                   , float a_Y
+                                   , float a_Z
+                                   , const Color& a_Color
+                                   , size_t timeStringLength
```

Comment:
The argument name "timeStringLength" could be generalized.  We could use the same mechanism to prioritize trailing text that is not necessarily time.

Reply:
Good point. How about... `a_TrailingCharsLength` ?

Reply:
Yeah perfect!